Makefile

```diff
@@ -94,11 +94,11 @@ cmd/bin/subctl: cmd/bin/subctl-$(VERSION)-$(GOOS)-$(GOARCH)$(GOEXE)
 
 dist/subctl-%.tar.xz: cmd/bin/subctl-%
 	mkdir -p dist
-	tar -cJf $@ --transform "s|^cmd/bin/subctl-[^/]*|subctl-$(VERSION)/subctl|" $<
+	tar -cJf $@ --transform "s|^cmd/bin/subctl-[^/]*|subctl-$(VERSION)/subctl|" --add-file=LICENSE $<
```

Comment:
The `LICENSE` file ends up being a prerequisite for the archive, list it as such and change the recipe to include all prerequisites:

```make
dist/subctl-%.tar.xz: cmd/bin/subctl-% LICENSE
	mkdir -p dist
	tar -cJf $@ --transform "s|^cmd/bin/subctl-[^/]*|subctl-$(VERSION)/subctl|" $^
```